pkg/data/management/machinedriver_data.go

```diff
@@ -115,6 +117,10 @@ func addMachineDrivers(management *config.ManagementContext) error {
 		"https://packethost.github.io/ui-driver-packet/1.0.2/component.js", "e03c6bc9406c811e03e9bc2c39f43e6cc8c02d1615bd0e0b8ee1b38be6fe201c", []string{"api.packet.net", "packethost.github.io"}, false, false, management); err != nil {
 		return err
 	}
+	if err := addMachineDriver(PhoenixNAPDriver, "https://github.com/phoenixnap/docker-machine-driver-pnap/releases/download/v0.1.0/docker-machine-driver-pnap_0.1.0_linux_amd64.zip",
+		"", "5f25a7fbcaca0710b7290216464ca8433fa3d683b59d5e4e674bef2d0a3ff6c7", []string{"api.securedservers.com"}, true, false, management); err != nil {
```

Comment:
The `active` value needs to be `false`, only the default ones are active on startup